File: WebSalesLibraries/protected/models/link_config_profile/models/LinkConfigProfileModel.php
<?php

	/**
	 * Class LinkConfigProfileModel
	 */
	class LinkConfigProfileModel
	{
		/**
		 * @var string id
		 * @soap
		 */
		public $id;
		/**
		 * @var string name
		 * @soap
		 */
		public $name;
		/**
		 * @var int order
		 * @soap
		 */
		public $order;
		/**
		 * @var LinkConfig config
		 * @soap
		 */
		public $config;
	}

File: SalesLibraries.ServiceConnector/Models/Soap/LinkConfigProfiles/LinkConfigProfileModel.cs
using System;

namespace SalesLibraries.ServiceConnector.LinkConfigProfileService
{
	public partial class LinkConfigProfileModel
	{
		public LinkConfigProfileModel()
		{
			InitDefaults();
		}

		private void InitDefaults()
		{
			id = Guid.NewGuid().ToString();

			config = new LinkConfig();
		
			config.disableDownload = false;
			config.disablePdf = false;
			config.disableFavorites = false;
			config.disableQuickSite = false;
			
			config.disablePreview = false;
			config.disableSave = false;
			config.disableEmail = false;

			config.libraryLinkTags = new string[] { };
			config.libraryReferences = new LibraryReference[] { };
			config.securityGroupReferences = new SecurityGroupReference[] { };
			config.ignoredLinkReferences = new LibraryLinkReference[] { };
		}
	}
}
